lib/zclient.c

```diff
@@ -3712,7 +3712,7 @@ static void zclient_event(enum event event, struct zclient *zclient)
 
 void zclient_interface_set_master(struct zclient *client,
 				  struct interface *master,
-				  struct interface *slave)
+				  struct interface *secondary)
```

Comment:
secondary does not capture what this is.  Please use `struct interface *member_intf`